pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -85,10 +85,19 @@ func CreateEmbeddedTarget(dep rv1beta2.DeploymentInterface, sa rv1.ServiceAccoun
 				dep.Delete(loggingconfig.EmbeddedESName, &metav1.DeleteOptions{})
 			}
 		}()
-		esDeployment := NewESDeployment(namespace)
+		esDeployment := NewESDeployment(namespace, obj)
 		_, err = dep.Create(esDeployment)
-		if err != nil && !apierrors.IsAlreadyExists(err) {
-			return errors.Wrapf(err, "create deployment %s fail", loggingconfig.EmbeddedESName)
+		if err != nil {
+			if !apierrors.IsAlreadyExists(err) {
+				return errors.Wrapf(err, "create deployment %s fail", loggingconfig.EmbeddedESName)
+			}
+			if err = updateEmbeddedQuota(dep, obj); err != nil {
```

Comment:
Nit: lets remove else cause, and just call `updateEmbeddedQuota(dep, obj);`. So the construct will be:

```
if err != nil && !apierrors.IsAlreadyExists(err) {
return errors.Wrapf(err, "create deployment %s fail", loggingconfig.EmbeddedESName)
}

if err = updateEmbeddedQuota(dep, obj); err != nil {
....
}
....
```

this is to avoid unnecessary code duplication